### DISCUSSION TITLE
Having problems with the Unauthorized.selector

### DISCUSSION THREAD

**Discussion by 0xV01** [foundry-rs/foundry#1547] [Help] [answered]
Hey guys,
I tried to follow along with the foundry book but I am stuck with this cheat code. Could someone please help me out?  I am sitting on this problem for 1 hr now trying to solve this issue. 

![image](https://user-images.githubusercontent.com/103319248/167252009-7c9a1e30-f6f9-4904-bcc7-876e8b95cbf7.png)

**Comment by 0xV01** [comment, marked as answer]
Edit: I am stupid and made a typo lol

**Reply by vinaywebcom** [reply]
Can you explain, I am facing the same issue!

**Reply by vinaywebcom** [reply]
Ohh so basically we need to pass the revert message instead of Unauthorized keyword lol
